script.js

```diff
@@ -0,0 +1,84 @@
+const mainMenu = document.querySelector('.mainMenu');
+const closeMenu = document.querySelector('.closeMenu');
+const openMenu = document.querySelector('.openMenu');
+
+function show() {
+  mainMenu.style.display = 'flex';
+  mainMenu.style.top = '0';
+}
+function close() {
+  mainMenu.style.top = '-120%';
+}
+
+openMenu.addEventListener('click', show);
+closeMenu.addEventListener('click', close);
+
+const featureLecturersList = [
+  {
```

Comment:
- [ ] Kindly change the design of the cards as the previous reviewer asked you to change to stick as much as you can to the design so please change it. Keep coding 💪 

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/163589068-efa6c74a-bb79-4724-b63c-815740a2f8df.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/163589122-828b5bf8-b8df-460b-81c9-295153ee2551.png)